wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/EnumType.kt

```diff
@@ -59,9 +60,30 @@ data class EnumType(
     if ("true" != allowAlias) {
       validateTagUniqueness(linker)
     }
+    validateTagNameAmbiguity("true" == allowAlias, linker)
     syntaxRules.validateEnumConstants(constants, linker.errors)
   }
 
+  private fun validateTagNameAmbiguity(allowAlias: Boolean, linker: Linker) {
+    val nameToConstants: Map<String, List<EnumConstant>> =
+        constants.groupBy { it.name.toEnglishLowerCase() }
```

Comment:
What the heck is English? I think somebody means ASCII? English is definitely not stable or formally specified.

Reply:
it uses `toLowerCase(Locale.US)` it's something we're already using in a few places.
I can do manual conversion here instead